docs/1.0/critical/index.md

```diff
@@ -1211,8 +1211,8 @@ The tradition is unambiguous, but the editor deems it useless and prints an emen
 ``` xml
 Utrum fides
 <app>
-  <lem type="conjecture-corrected">sit</lem>
-  <rdg wit="#P #V #L"><corr>servus</corr></rdg>
```

Comment:
Ha! This is funny. I didn't know that I didn't write that myself (of course I could have checked the history). But I must admit that I did find it a bit curious, but since it was in there I figured "well, we/I must have had a good reason for using this". And I assumed that that good reason was exactly what you point out, the parallel structure with the other conjectures.

It is right though that there are several indications that this is an editorial addition. 
1. The reading type states it pretty clearly, and
2. the absence of a `@wit` clearly points in that direction.

So I would not have any problem removing it.
The a question remains: Should we maintain the parallel elements on the other conjecture types (`<supplied>` and `<surplus>`)? It seems to me that we need to be consequent in this, and that either we keep the elements on all of them or we remove it on all. But maybe there are better reasons for keeping them on the other types?